### DISCUSSION TITLE
템플릿 업로드 오류

### DISCUSSION THREAD

**Discussion by miseong43** [naverz/zepeto-studio-kor#758] [5. Q&A]
![error](https://user-images.githubusercontent.com/90840958/168461590-47be9728-3f4a-4e91-8a2f-d450c98ff48e.JPG)

안녕하세요,

처음에 탬플릿 불러올때 이런 메세지가 떠서 yes 를 누르고 진행하였더니 오브젝트가 보이지 않습니다.

어떻게 하면될까요? 도와주세요~

**Comment by x4cf** [comment]
Project Settings ⇒ Player ⇒ Active Input Handling* 을 Both로 변경하면 팝업창이 나오지 않아요 :)
처음 시작할때는 No를 누르고 진입해주세요
<img width="694" alt="오류" src="https://user-images.githubusercontent.com/90679926/168463002-d4f95468-adba-40b4-ab37-e3fe92d68ff1.png">

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, 위 팝업 메세지는 유니티의 New Input system을 사용하겠냐는 뜻입니다.
ZEPETO 캐릭터 컨트롤은 V-Pad를 사용하는 대신 Unity의 NewInputSystem을 사용하도록 커스텀할 수 있기에 
가급적 Yes를 사용해 생성 부탁 드리며 

처음에 오브젝트가 나오지 않는 것은 Sample Scene을 더블클릭하면 나올 것으로 예상됩니다.
감사합니다.
